docs/releasehistory.rst

```diff
@@ -75,23 +75,23 @@ New features
 - `PR #548 <https://github.com/openforcefield/openforcefield/pull/548>`_: Adds ``replace`` and ``all_permutations`` kwarg to
 
   - :py:meth:`Molecule.add_bond_charge_virtual_site <openforcefield.topology.Molecule.add_bond_charge_virtual_site>`
-  - :py:meth:`Molecule.add_monovalent_virtual_site <openforcefield.topology.Molecule.add_monovalent_virtual_site>`
-  - :py:meth:`Molecule.add_divalent_virtual_site <openforcefield.topology.Molecule.add_divalent_virtual_site>`
-  - :py:meth:`Molecule.add_trivalent_virtual_site <openforcefield.topology.Molecule.add_trivalent_virtual_site>`
+  - :py:meth:`Molecule.add_monovalent_lone_pair_virtual_site <openforcefield.topology.Molecule.add_monovalent_lone_pair_virtual_site>`
+  - :py:meth:`Molecule.add_divalent_lone_pair_virtual_site <openforcefield.topology.Molecule.add_divalent_lone_pair_virtual_site>`
+  - :py:meth:`Molecule.add_trivalent_lone_pair_virtual_site <openforcefield.topology.Molecule.add_trivalent_lone_pair_virtual_site>`
 
 - `PR #548 <https://github.com/openforcefield/openforcefield/pull/548>`_: Adds ``orientations`` to
 
-  - :py:class:`BondChargeVirtualSite <openforcefield.topology.molecule.BondChargeVirtualSite>` 
-  - :py:class:`MonovalentLonePairVirtualSite <openforcefield.topology.molecule.MonovalentLonePairVirtualSite>` 
-  - :py:class:`DivalentLonePairVirtualSite <openforcefield.topology.molecule.DivalentLonePairVirtualSite>` 
-  - :py:class:`TrivalentLonePairVirtualSite <openforcefield.topology.molecule.TrivalentLonePairVirtualSite>` 
+  - :py:class:`BondChargeVirtualSite <openforcefield.topology.BondChargeVirtualSite>`
+  - :py:class:`MonovalentLonePairVirtualSite <openforcefield.topology.MonovalentLonePairVirtualSite>`
+  - :py:class:`DivalentLonePairVirtualSite <openforcefield.topology.DivalentLonePairVirtualSite>`
+  - :py:class:`TrivalentLonePairVirtualSite <openforcefield.topology.TrivalentLonePairVirtualSite>`
 
 - `PR #548 <https://github.com/openforcefield/openforcefield/pull/548>`_: Adds
 
-  - :py:class:`VirtualParticle <openforcefield.topology.molecule.VirtualParticle>`
-  - :py:class:`TopologyVirtualParticle <openforcefield.topology.topology.TopologyVirtualParticle>`
-  - :py:class:`DuplicateVirtualSiteTypeException <openforcefield.typing.engines.smirnoff.parameters.DuplicateVirtualSiteTypeException>`
-  - :py:meth:`TopologyVirtualParticle.virtual_particle_start_topology_index <openforcefield.topology.molecule.TopologyVirtualParticle.virtual_particle_start_topology_index>`
+  - :py:class:`VirtualParticle <openforcefield.topology.VirtualParticle>`
+  - :py:class:`TopologyVirtualParticle <openforcefield.topology.TopologyVirtualParticle>`
+  - :py:class:`DuplicateVirtualSiteTypeException <openforcefield.typing.engines.smirnoff.DuplicateVirtualSiteTypeException>`
```

Comment:
I can't get this link to be populated - I think this would require a section in `typing.rst`/`typing.html` dedicated just to hosting the exceptions. But I don't really understand sphinx well enough to be sure

Reply:
I've gone through these today (with your help for quite some time!) and wired them up correctly. There were some issues where Sphinx didn't know to remove the previous contents of `docs/api/`, even after running `make clean`, and others where, just due to the sheer size of the VSites PR, a few public methods ended up without docstrings so the API links would fail. 

I think I've filled in most of the cracks in the new method docstrings :-)